include/openssl/trace.h

```diff
@@ -49,14 +49,36 @@ extern "C" {
 # define OSSL_TRACE_CATEGORY_PKCS12_DECRYPT     10
 # define OSSL_TRACE_CATEGORY_X509V3_POLICY      11
 # define OSSL_TRACE_CATEGORY_BN_CTX             12
-# define OSSL_TRACE_CATEGORY_NUM                13
+# define OSSL_TRACE_CATEGORY_CMP                13
+# define OSSL_TRACE_CATEGORY_NUM                14
 
 /* Returns the trace category number for the given |name| */
 int OSSL_trace_get_category_num(const char *name);
 
 /* Returns the trace category name for the given |num| */
 const char *OSSL_trace_get_category_name(int num);
 
+/*
+ * TRACE LEVEL AND VERBOSITY
+ */
+
+# define OSSL_TRACE_LEVEL_UNDEFINED     0
```

Comment:
I see no use in having an 'undefined' trace level. Normally, I would call it 'off' or 'disabled', but since trace categories are enabled by attaching channels to them, a 'disabled' level does not make sense IMHO.